server.js

```diff
@@ -24,7 +16,40 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Endpoints: /books, /books/id')
```

Comment:
If you want to make it looks nicer, you can create an HTML file and serve it to the user :) 
Maybe you could add info about available queries here? :)